### ISSUE TITLE
develop ml based turning point detection model

### ISSUE THREAD

**Issue by haidaoxiaofei** [haidaoxiaofei/GPS2RoadNetwork#6]
1. Use the existing map to label the existing turning points on the map
2. Training model based on the tracks and the labelled turning points
3. Test the trained model on raw tracks.

**Comment by haidaoxiaofei** [comment]
Basically, we will test the simple models, like LR, first.
1. Using the existing map to extract the ground truth of the intersections.
2. Extract features (e.g., speed, direction change, time-window features) from trajectories
3. build a simple model with xgboost to see the effectiveness of detecting intersections.
